charts/airflow/files/pod_template.kubernetes-helm-yaml

```diff
@@ -38,14 +38,17 @@ spec:
   securityContext:
     {{- $podSecurityContext | nindent 4 }}
   {{- end }}
-  {{- if or ($extraPipPackages) (.Values.dags.gitSync.enabled) }}
+  {{- if or ($extraPipPackages) (.Values.dags.gitSync.enabled) (.Values.kubernetesPodTemplate.extraInitContainers) }}
```

Comment:
This is incorrect, the value is `airflow.kubernetesPodTemplate.extraInitContainers`, not `kubernetesPodTemplate.extraInitContainers`.